src/constants/index.js

```diff
@@ -31,6 +31,9 @@ export const PROVISION_SOURCE_TEMPLATE = 'Template';
 
 export const POD_NETWORK = 'Pod Networking';
 
+export const ANNOTATION_PXE_INTERFACE = 'cnv.ui.pxeInterface';
```

Comment:
nitpick: Is this constant explicitly defined somewhere in documentation (or so)?
Does it make sense to add reference link as a comment here?

Same as the constants above.

Reply:
no, I just made it up. This is used only by us (UI) for later VM edit

Reply:
For other constants..some of them are used in common-templates, but there are no docs, we could link to the common-templates repo ..?
Others are used in kubevirt itself (like API_VERSION)..we could link to user guide's VM example.. but that does not seem correct..